config/dev.webpack.config.js

```diff
@@ -64,7 +57,7 @@ const webpackProxy = {
 
 const { config: webpackConfig, plugins } = config({
   rootFolder: resolve(__dirname, '../'),
-  sassPrefix: '.hacDev',
+  sassPrefix: '.hacCore',
```

Comment:
I don't think this change is intended...

Reply:
Well the thing is that console.dot is wrapping each app with its prefix. Since this is not directly an app, but extension it inherits the prefix of `hacCore` but I guess we could update it as `.hacCore, .hacDev`. Or (which might be better imo) wrapping each `route` with `Container` with the app's name in `hac-core`

Reply:
Opened a PR in hac-core to allow passing className as wrapper of current route. This way one route styling won't affect styling of different route.